lib/Assistance.dart

```diff
@@ -0,0 +1,121 @@
+
+class Assistance extends StatelessWidget {
+
+  final cancelStyle = TextStyle(
+    color: Colors.black,
+    fontWeight: FontWeight.w100,
+    fontSize: 15,
+  );
+
+  final questionStyle = TextStyle(
+    color: Colors.black,
+    fontWeight: FontWeight.w800,
+    fontSize: 25,
+  );
+
+  final toggleStyle = TextStyle(
+    fontWeight: FontWeight.w200,
+    fontSize: 18,
+  );
+
+  final descriptionStyle = TextStyle(
+      color: Colors.grey,
+      fontWeight: FontWeight.w100,
+      fontSize: 13
+  );
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Container(
+        margin: EdgeInsets.only(top: 50.0, left: 20.0, right: 20.0),
+        child: Column(
+          children: <Widget>[
+            Row(
+              mainAxisAlignment: MainAxisAlignment.start,
+              children: <Widget>[
+                Container(
+                  child: InkWell(
+                    child: Text("Cancel", style: cancelStyle),
+                    onTap: () {
+                      Navigator.pop(context, new MaterialPageRoute(builder: (context) => RepeatRide()));
+                    },
+                  ),
+                ),
+              ],
+            ),
+            SizedBox(height: 50.0),
+            Row(
+              mainAxisAlignment: MainAxisAlignment.start,
+              children: <Widget>[
+                Padding(
+                  padding: const EdgeInsets.only(right: 4.0),
+                  child: Icon(Icons.brightness_1, size: 12.0),
+                ),
+                Padding(
+                  padding: const EdgeInsets.only(right: 4.0),
+                  child: Icon(Icons.brightness_1, size: 12.0),
+                ),
+                Padding(
+                  padding: const EdgeInsets.only(right: 4.0),
+                  child: Icon(Icons.brightness_1, size: 12.0),
+                ),
+                Padding(
+                  padding: const EdgeInsets.only(right: 4.0),
+                  child: Icon(Icons.group_work),
+                ),
+                Icon(Icons.brightness_1, color: Colors.grey[350], size: 12.0),
+              ],
+            ),
+            SizedBox(height: 10.0),
+            Row(
+              children: <Widget>[
+                Flexible(child: Text("Do you need any assistance? (optional)", style: questionStyle))
+              ],
+            ),
+            SizedBox(height: 80),
+            Row(
+              mainAxisAlignment: MainAxisAlignment.start,
+              children: <Widget>[
+                Icon(Icons.panorama_fish_eye),
+                SizedBox(width: 10.0),
+                Text('Wheelchair')
+              ],
+            ),
+            SizedBox(height: 20),
+            Row(
+              mainAxisAlignment: MainAxisAlignment.start,
+              children: <Widget>[
+                Icon(Icons.panorama_fish_eye),
+                SizedBox(width: 10.0),
+                Text('Other')
+              ],
+            ),
```

Comment:
Instead of hardcoding this, use radio buttons. https://api.flutter.dev/flutter/material/Radio-class.html